docker-compose.yml

```diff
@@ -17,8 +16,7 @@ services:
     ports:
       - "6306:3306"
     volumes:
-      - ./.data/mysql:/var/lib/mysql:delegated
-      - ./mysql:/docker-entrypoint-initdb.d:ro
```

Comment:
I think we need to leave this in here otherwise the mysql/init.sql won't run on initial startup. This caused an issue for Yong recently. I'm not sure why we didn't need that for our other databases.

Reply:
It seems to be working fine for me without this, but it's taking a long time to startup and creating almost 200MB of large files. Though looking back at 5.7 it also created large ib_logfile0 and ib_logfile1, but no other large files. so took up about half the space. Maybe it's not worthwhile to fix here.

I'd `kind of` like to get all of our T&L Docker databases to use the same image and name. This would make it a little easier to work with locally as the number of apps we support grow.

I guess this is a different issue though.

Reply:
Thanks for reviewing. I will commit the PR for now.